native/dll/Win32DLL/SageTVWin32DLL.cpp

```diff
@@ -1013,7 +1021,7 @@ void loadAWTLib()
 		*goodSlash = 0;
 		goodSlash = strrchr(jvmPath, '\\');
 		if (!goodSlash) return;
-		strcpy(goodSlash + 1, "jawt.dll");
+		strcpy_s(jvmPath, sizeof(jvmPath), "jawt.dll");

```

Comment:
Look at my prior comments here....with what you have now you just end up with 'jawt.dll' for the jvmPath....nothing is being done with all the slash stuff.

Reply:
Done